notebooks/error_analysis.ipynb

```diff
@@ -246,7 +246,10 @@
    "outputs": [],
    "source": [
     "def plot_tif(tif_name, start_date, inst_lon, inst_lat, crop_prob, max_val=2000):\n",
-    "    tif = load_tif(tifs_dir / tif_name, start_date, 30)\n",
+    "    \n",
+    "    raise NotImplementedError(\"Tif now needs to be downloaded first\")\n",
+    "    \n",
+    "    #tif = load_tif(tifs_dir / tif_name, start_date, 30)\n",
```

Comment:
The notebook used to use the tifs available locally, but if it is to be used again they will have to be pulled in from the cloud